### ISSUE TITLE
Issue making sequences predictions by CompGraph (BasicRNNExample)

### ISSUE THREAD

**Issue by samuelchapas** [deeplearning4j/deeplearning4j#5133]
After making a lot of trial, we have checked every detail in code and for a training with 1000 epoch, we couldn't get a characters prediction very fit, we only used one word but after several epoch it doesn't seem a representative sequence being predicted. We will be thankfull for any help. The next is a sample of the log by training the model with the word "Postkutscher".

Repository: [Code](https://github.com/samuelchapas/LSTMEncDecDL4J/tree/master/src/main/java/flyKey)

```
Epoch 992
22:11:14,477 INFO  ~ Score at iteration 992 is 62.91703153856858
rrrrrrrrrrrr
Epoch 993
22:11:14,578 INFO  ~ Score at iteration 993 is 63.0912701078713
rrrrrrrrrrrr
Epoch 994
22:11:14,682 INFO  ~ Score at iteration 994 is 63.135271460716915
rrrrrrrrrrrr
Epoch 995
22:11:14,781 INFO  ~ Score at iteration 995 is 63.30987945806829
rrrrrrrrrrrr
Epoch 996
22:11:14,883 INFO  ~ Score at iteration 996 is 63.35328218441111
rrrrrrrrrrrr
Epoch 997
22:11:14,982 INFO  ~ Score at iteration 997 is 63.528359867998184
rrrrrrrrrrrr
Epoch 998
22:11:15,085 INFO  ~ Score at iteration 998 is 63.572412734310625
rrrrrrrrrrrr
Epoch 999
22:11:15,187 INFO  ~ Score at iteration 999 is 63.74762266711256
rrrrrrrrrrrr
```

**Comment by AlexDBlack** [comment]
Usually those issues are related to tuning - selection of hyperparameters (layer size, learning rate, etc).
I'd suggest taking a look at the following pages:
https://deeplearning4j.org/visualization
https://deeplearning4j.org/troubleshootingneuralnets
